src/base/utils/os.cpp

```diff
@@ -87,11 +85,9 @@ void Utils::OS::shutdownComputer([[maybe_unused]] const ShutdownDialogAction &ac
     }
     else
     {
-        const QString msg = QCoreApplication::translate("misc"
-            , "qBittorrent will shutdown the computer now because all downloads are complete.");
-        auto msgWchar = std::make_unique<wchar_t[]>(msg.length() + 1);
-        msg.toWCharArray(msgWchar.get());
-        ::InitiateSystemShutdownW(nullptr, msgWchar.get(), 10, TRUE, FALSE);
+        std::wstring msg = QCoreApplication::translate("misc"
```

Comment:
```suggestion
        const std::wstring msg = QCoreApplication::translate("misc"
```

Reply:
This can't be `const` due to `InitiateSystemShutdownW` doesn't take a const pointer to string.
https://learn.microsoft.com/en-us/windows/win32/api/winreg/nf-winreg-initiatesystemshutdownw